docs/book/src/quick-start.md

```diff
@@ -27,7 +27,7 @@ arch=$(go env GOARCH)
 curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
 ```
 
-If you are using a Kubebuilder plugin version less than version `v3+`, you must configure the Kubernetes binaries required for the [envtest][envtest], run: 
+If you are using a Kubebuilder CLI version =< `v2.3.1`, you must configure the Kubernetes binaries required for the [envtest][envtest], run: 
```

Comment:
Don't you need to do the steps below regardless of kubebuilder version, since the kubebuiler binary itself is in that tarball?

Reply:
These steps are required currently/today. So, we need to keep the steps to let the users know what is required to do for versions until the current one.

Reply:
Agreed. I am saying that you'll need to run these steps no matter what version you're using. The tool binaries haven't yet been removed from the release tarball, and the release tarball contains the kubebuiler binary itself.
```suggestion
To add `kubebuilder` and [envtest][envtest] binaries to your path: 
```